src/comonents/ButtonPage.jsx

```diff
@@ -1,9 +1,11 @@
-import React from 'react'
+import React, { useContext } from 'react'
+import { MyNoteContext } from '../context/NoteState'
 
 function ButtonPage() {
+  const {handleClick}=useContext(MyNoteContext);
   return (
     <div>
-      
+      <button style={{background:"#2222c9",color:"#ffffff"}} onClick={handleClick}>Submit</button>
```

Comment:
remove inline css, create seperate file for css and attach

Reply:
Changed sir